code/modules/clothing/spacesuits/void/void_vr.dm

```diff
@@ -368,3 +368,21 @@
 			else
 				H << "<span class='warning'>You need to have a horse, wolf, or naga half to wear this.</span>"
 				return 0
+
+/obj/item/clothing/suit/space/void/security/hasd
```

Comment:
Why is this not in custom_clothes_vr.dm? It's ckey locked, so it definitely should be.

Reply:
Ack, let me move that. Forgot about that.